contracts/script/avs/DeployGoerliAVS.s.sol

```diff
@@ -22,52 +18,77 @@ import { Script } from "forge-std/Script.sol";
  *      deploy script.
  */
 contract DeployGoerliAVS is Script {
-    uint96 public constant MIN_OPERATOR_STAKE = 1 ether;
-    uint32 public constant MAX_OPERATOR_COUNT = 10;
+    uint64 internal _omniChainId = 165;
+    address internal _create3Factory = address(uint160(uint256(keccak256("TODO"))));
+    address internal _proxyAdmin = address(uint160(uint256(keccak256("TODO"))));
+    address internal _owner = address(uint160(uint256(keccak256("TODO"))));
+    address internal _portal = address(0);
+    address internal _ethStakeInbox = address(0);
+    bytes32 internal _create3Salt = keccak256("TODO");
 
     /// @dev forge script entrypoint
-    function run() public pure {
-        revert("Not implemented");
+    function run() public {
+        uint64 omniChainId = uint64(vm.envUint("OMNI_CHAIN_ID"));
+        uint256 deployerKey = vm.envUint("DEPLOYER_KEY");
+        address factory = vm.envOr("CREATE3_FACTORY_ADDRESS", _create3Factory);
+        address proxyAdmin = vm.envOr("PROXY_ADMIN", _proxyAdmin);
+        address owner = vm.envOr("AVS_OWNER", _owner);
+        address portal = vm.envOr("PORTAL_ADDRESS", _portal);
+        address ethStakeInbox = vm.envOr("ETH_STAKE_INBOX", _ethStakeInbox);
+        bytes32 salt = vm.envOr("CREATE3_SALT", _create3Salt);
+
+        vm.startBroadcast(deployerKey);
+        deploy(factory, salt, owner, proxyAdmin, portal, omniChainId, ethStakeInbox);
```

Comment:
so we''ll need to have deployed a proxy admin first?

Reply:
Yes adding a script for that in separate Pr